styles/components/_button.scss

```diff
@@ -0,0 +1,21 @@
+.button__container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  top: 2px;
+}
+
+.button {
```

Comment:
Add border to the button, right doesn't look flat

Reply:
Done !